apparmor/loader/example-configmap.yaml

```diff
@@ -0,0 +1,76 @@
+# An example ConfigMap demonstrating how profiles can be stored as Kubernetes objects, and loaded by
+# the DaemonSet.
```

Comment:
Done. I went with "apparmor-loader DaemonSet" since that's the actual object name.